applications/MultilevelMonteCarloApplication/python_scripts/adaptive_refinement_utilities.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+from copy import copy
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.MeshingApplication as MeshingApplication
+
+# Import cpickle to pickle the serializer
+try:
+    import cpickle as pickle  # Use cPickle on Python 2.7
```

Comment:
just curious, does this work for the c-objects in Kratos?

Reply:
i added pickling for 2 objects: model and serialization which happen to be "self contained". It is less flexible that constructing the serialization object and we should NOT add to all of the objects